src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,175 @@
+package duke;
+
+import duke.task.*;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.File;
+public class Duke {
+    public static void printGreeting(){
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+    }
+
+    public static void printFarewell(){
+        System.out.println("Bye. Hope to see you again soon!");
+    }
```

Comment:
Minor coding standard issue, but consider standardizing the spacing between brackets and braces, in adherence to coding standard.
This issue is present a couple more times throughout `Duke.java`, so do keep a look out and try to make it consistent :)